nextcord/member.py

```diff
@@ -249,7 +249,8 @@ class Member(abc.Messageable, _UserTag):
     if TYPE_CHECKING:
         name: str
         id: int
-        discriminator: str
+        global_name: Optional[str]
```

Comment:
This is a breaking change. It is still typed as `str` in the API. Leave it as the compatible `"0"` for now.